ophyd/signal.py

```diff
@@ -1474,18 +1573,14 @@ def describe(self):
         dict
             Dictionary of name and formatted description string
         """
-        if self._readback is DEFAULT_EPICSSIGNAL_VALUE:
-            val = self.get()
-        else:
-            val = self._readback
+        desc = super().describe()[self.name]
```

Comment:
If `super().describe()` happens to return more than one key this will silently drop the other keys on the ground.

I don't think it _should_ do that (and if it were we would currently not even be asking the parent class to source a describe method), but it seems better to be defensive and either do

```python
ret = super().describe()
desc = ret[self.name]
...
return ret
```

or

```python
(name, desc), = super().describe().items()
if name != self.name: raise RuntimeError
```

Reply:
Changed (using first proposal), both in `Signal.describe` and `DerivedSignal.describe`